pkg/util/helpers_test.go

```diff
@@ -77,7 +77,7 @@ func TestCoerceTypeJSONMarshalerToByteArray(t *testing.T) {
 	assert.IsType(t, reflect.TypeOf(expectedType), reflect.TypeOf(result))
 }
 func TestCoerceTypeNotSupportedToByteArray(t *testing.T) {
-	myData := 25
+	myData := make(chan int)
```

Comment:
Please comment why using a channel.

Reply:
Channels are not marshalable to JSON and are the standard way to generate that error.